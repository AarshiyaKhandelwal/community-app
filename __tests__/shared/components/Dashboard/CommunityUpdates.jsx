import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import CommunityUpdates from 'components/Dashboard/CommunityUpdates';

const blogsData = [{ link: 'http', title: 'title', description: 'description' }];

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  renderer.render((
    <CommunityUpdates blogs={blogsData} />
  ));
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
