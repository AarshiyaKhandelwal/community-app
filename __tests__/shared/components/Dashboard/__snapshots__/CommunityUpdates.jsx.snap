// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div
  className="src-shared-components-Dashboard-CommunityUpdates-___styles__community-updates___avu0R"
>
  <header>
    <h1
      className="src-shared-components-Dashboard-CommunityUpdates-___styles__section-title___3c5vb"
    >
      From the Community Blog
    </h1>
  </header>
  <div
    className="src-shared-components-Dashboard-CommunityUpdates-___styles__posts___30jO_"
  >
    <div
      className="src-shared-components-Dashboard-CommunityUpdates-___styles__post___3Ym8I"
    >
      <div
        className="src-shared-components-Dashboard-CommunityUpdates-___styles__blog-link___3G-rk"
      >
        <a
          href="http"
          rel="noopener noreferrer"
          target="_blank"
          title="title"
        >
          title
        </a>
      </div>
      <div
        className="src-shared-components-Dashboard-CommunityUpdates-___styles__description___1xI5M"
        dangerouslySetInnerHTML={
          Object {
            "__html": "description",
          }
        }
      />
    </div>
  </div>
  <div
    className="src-shared-components-Dashboard-CommunityUpdates-___styles__blog-links___2swU9"
  >
    <a
      href="https://www.topcoder-dev.com/blog/"
    >
      View More
    </a>
  </div>
</div>
`;
